Signal/src/ViewControllers/OWSSoundSettingsViewController.h

```diff
@@ -6,10 +6,7 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-typedef NS_ENUM(NSUInteger, OWSSoundType) {
-    OWSSoundType_Notification = 0,
-    OWSSoundType_Ringtone,
-};
+typedef NS_ENUM(NSUInteger, OWSSoundType) { OWSSoundType_Notification = 0 };
```

Comment:
Does this enum make sense anymore?

We now have switches that only have one case - getting rid of it seems like a nice little cleanup.

Reply:
Done.  I was thinking we might use it again...